src/components/calendar/ClassModal.js

```diff
@@ -80,7 +80,11 @@ export default function ClassModal({ showDialog, toggleDialog, section, associat
           {section.descriptions.map(description => (
             <div key={JSON.stringify(description)}>
               <DialogContentText className={classes.noBorder} variant="h6">{description.name}</DialogContentText>
-              <DialogContentText>{description.value}</DialogContentText>
+              {description.value.split('<br/>').map(description2 => (
```

Comment:
I don't like naming the variable `description2`, maybe `paragraph` would be better here.